src/app/other-user-profiles/other-user-profiles.component.html

```diff
@@ -7,9 +7,12 @@ <h1>Other user skill profiles</h1>
   <mat-card *ngFor="let user of permissionOwners" class="other-user-profiles-card">
     <mat-card-content>
       <div fxLayout="row">
-        <div fxFlex>
+        <div fxFlex="90%">
           <h2 class="other-user-profiles-name">{{ user.userName }}</h2>
         </div>
+        <div fxFlex="10%">
```

Comment:
And let's remove here `fxFlex="10%"` at all. The div will be row-placed because of parent's `fxLayout="row"` and its size should be adjusted automatically.

Reply:
done